packages/lab/src/chat/Chat.tsx

```diff
@@ -0,0 +1,45 @@
+// Chat.tsx
+import { Typography } from '@material-ui/core';
+import { Stack } from '@superdispatch/ui';
+import { forwardRef } from 'react';
+import styled from 'styled-components';
+
+interface ChatProps {
+  children?: React.ReactNode;
+}
+
+const ChatContainer = styled('div')<{
+  isEmpty: boolean;
+}>(
+  ({ isEmpty }) => `
+  display: flex;
+  flex-direction: column;
+  overflow: auto;
+  justify-content: ${isEmpty ? 'center' : 'flex-end'};
+  width: 100%;
+  height: 100%;
+  min-height: 368px;
+  max-height: 696px;
+  max-width: 560px;
+  min-width: 432px;
+  padding: 16px;
+  box-sizing: border-box;
+`,
+);
+
+export const Chat = forwardRef<HTMLDivElement, ChatProps>(({ children }) => {
+  const emptyPlaceholder = (
+    <Typography color="textSecondary" align="center">
+      No new messages from Super Shipper. <br />
+      We will let you know when they send you a message.
+    </Typography>
+  );
+
+  return (
+    <ChatContainer data-testid="chat-container" isEmpty={!children}>
+      <Stack space="small">{!children ? emptyPlaceholder : children}</Stack>
+    </ChatContainer>
+  );
```

Comment:
@TravnikovDev Can we also extract `emptyPlaceholder` as a standalone component or move it inside the condition itself? So it will render only if the chat is empty.

Reply:
Moved